anitya/lib/backends/gnome.py

```diff
@@ -58,7 +84,12 @@ def get_versions(cls, project):
             when the versions cannot be retrieved correctly
 
         '''
-        url = 'https://download.gnome.org/sources/%(name)s/' % {
-            'name': project.name}
 
-        return get_versions_by_regex(url, REGEX, project)
+        output = []
+        try:
+            # First try to get the version by using the cache.json file
+            output = use_gnome_cache_json(project)
+        except Exception:
```

Comment:
Can we log a warning here about the details of the exception?  Otherwise, if the cache/json stuff breaks, we'll never know.